commands/ft.create.md

```diff
@@ -0,0 +1,83 @@
+The `FT.CREATE` command creates an empty index and initiates the backfill process. Each index consists of a number of field definitions. Each field definition specifies a field name, a field type and a path within each indexed key to locate a value of the declared type. Some field type definitions have additional sub-type specifiers.
+
+For indexes on HASH keys, the path is the same as the hash member name. The optional `AS` clause can be used to rename the field if desired.
+Renaming of fields is especially useful when the member name contains special characters.
+
+For indexes on JSON keys, the path is a JSON path to the data of the declared type. Because the JSON path always contains special characters, the `AS` clause is required.
+
+
+```
+FT.CREATE <index-name>
+    ON HASH | JSON
```

Comment:
```suggestion
    [ON HASH | JSON]
```
You say these are optional. This will also automatically be generated based on the json files in the search repository, so do we need a secondary example here?